### PR TITLE
Add motivation to apply, and finalize submission forms

### PR DESCRIPTION
**Pull request by Nebrethar**

### CODE REVIEW

**Comment by Nebrethar** [comment]
A note: We are dynamically generating the badge SVGs so that really big link is necessary. That's there because we are including the CHAOSS logo in SVG

**Comment by Nebrethar** [comment]
Requesting review from @germonprez @klumb on the added motivation section

File: Programs/DIBadging/description.md [hunk 1]
```diff
@@ -3,12 +3,24 @@ Thank you for showing interest in D&I Badging for your event! The Event Badging
...
+
+<img src="https://img.shields.io/badge/D%26I-Badging-passing?style=flat-square&labelColor=583586&logo=data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDI1MCAyNTAiPgo8cGF0aCBmaWxsPSIjMUM5QkQ2IiBkPSJNOTcuMSw0OS4zYzE4LTYuNywzNy44LTYuOCw1NS45LTAuMmwxNy41LTMwLjJjLTI5LTEyLjMtNjEuOC0xMi4yLTkwLjgsMC4zTDk3LjEsNDkuM3oiLz4KPHBhdGggZmlsbD0iIzZBQzdCOSIgZD0iTTE5NC42LDMyLjhMMTc3LjIsNjNjMTQuOCwxMi4zLDI0LjcsMjkuNSwyNy45LDQ4LjVoMzQuOUMyMzYuMiw4MC4yLDIxOS45LDUxLjcsMTk0LjYsMzIuOHoiLz4KPHBhdGggZmlsbD0iI0JGOUNDOSIgZD0iTTIwNC45LDEzOS40Yy03LjksNDMuOS00OS45LDczLTkzLjgsNjUuMWMtMTMuOC0yLjUtMjYuOC04LjYtMzcuNS0xNy42bC0yNi44LDIyLjQKCWM0Ni42LDQzLjQsMTE5LjUsNDAuOSwxNjIuOS01LjdjMTYuNS0xNy43LDI3LTQwLjIsMzAuMS02NC4ySDIwNC45eiIvPgo8cGF0aCBmaWxsPSIjRDYxRDVGIiBkPSJNNTUuNiwxNjUuNkMzNS45LDEzMS44LDQzLjMsODguOCw3My4xLDYzLjVMNTUuNywzMy4yQzcuNSw2OS44LTQuMiwxMzcuNCwyOC44LDE4OEw1NS42LDE2NS42eiIvPgo8L3N2Zz4K" height=48 /><br>
+
+Applying for a badge supports the open source community as a whole by expressing that your event is willing to improve. This ultimately affects D&I practices in your event and even outside of your project space.
```
Comment (germonprez) [inline Programs/DIBadging/description.md:12]: Maybe: 

Applying for a badge supports D&I efforts within an open source community by expressing that your event is willing to improve the ways the work. These efforts can have affects to D&I practices in your event and even outside of your project space.

**Review by germonprez — COMMENTED** [review]
Looks good. Minor structure suggestion.

**Comment by klumb** [comment]
@Nebrethar Sorry, I was a little premature in merging this.

**Pull request merged**